### PR TITLE
Actor tests

### PR DESCRIPTION
**Pull request by Michal-Suhak**

### CODE REVIEW

File: src/quokka_editor_back/actors/task.py [hunk 1]
```diff
@@ -23,6 +22,7 @@
 
 
 def decode_document_content(document):
+    # TODO The `b""` value will raise an error.
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/actors/task.py:25]: so what should be returned then?

File: src/quokka_editor_back/actors/task.py [hunk 2]
```diff
@@ -51,16 +51,15 @@ async def process_one_operation(
...
-async def process_operations(
-    redis_client: AsyncRedis, document_id: str, document: Document
-) -> None:
-    async for op_data in fetch_operations_from_redis(redis_client, document_id):
+async def process_operations(redis_client: AsyncRedis, document: Document) -> None:
+    async for op_data in fetch_operations_from_redis(redis_client, document.id):
         loaded_op_data = json.loads(op_data)
         user_token = loaded_op_data["user_token"]
         if new_op := await process_one_operation(loaded_op_data, document):
+            breakpoint()
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/actors/task.py:59]: :O

File: src/quokka_editor_back/tests/actors/test_tasks.py [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "content, desired_value",
+    [
+        (json.dumps({"key": "value"}).encode(), {"key": "value"}),
+        # (b"", {}),
+        # (b"invalid_json_data", {}),
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/tests/actors/test_tasks.py:38]: why it's commented?

File: src/quokka_editor_back/actors/task.py (hunk 1 above)
Comment (DamianCzajkowski) [reply src/quokka_editor_back/actors/task.py]: dict

File: src/quokka_editor_back/tests/actors/test_tasks.py [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+    async def __anext__(self):
+        if self.index < len(self.data_list):
+            value = self.data_list[self.index]
+            self.index += 1
+            return value
+        else:
+            raise StopAsyncIteration
+
+
+# TODO should be fixed.
```
[the remark this answers is not in the document]
Comment (DamianCzajkowski) [reply src/quokka_editor_back/tests/actors/test_tasks.py:121]: Is this fixed already?

[the remark this answers is not in the document]
Comment (Michal-Suhak) [reply src/quokka_editor_back/tests/actors/test_tasks.py:121]: Tests should be fixed, but I'm too stupid to do this

[the remark this answers is not in the document]
Comment (Michal-Suhak) [reply src/quokka_editor_back/tests/actors/test_tasks.py]: yes

**Review by DamianCzajkowski — APPROVED** [review]

**Pull request merged**